samples/MusicStore/MusicStore.Test/Controllers/StoreManagerControllerTest.cs

```diff
@@ -0,0 +1,328 @@
+﻿using Microsoft.AspNetCore.Mvc;
```

Comment:
Using should be in the namespace, using short syntax. System on top, sorted alphabetically.